rfd/0103-application-access-auth-flow.md

```diff
@@ -0,0 +1,76 @@
+---
+authors: Ryan Clark (ryan.clark@goteleport.com)
+state: draft
+---
+
+# RFD 103 - Application Access Auth Flow
+
+## What
+
+This is an overview of the flow used for setting the relevant cookies when a user logs in to an 
+
+application through Teleport.

```

Comment:
```suggestion
application through Teleport Web UI.
```
nit

Reply:
Cheers @GavinFrazar - I've committed this, do you think we should differentiate more that this is in the web UI or is mentioning it here enough?

Reply:
It may be useful for finding this document to rename the RFD document as well like `rfd/0103-application-access-web-ui-auth-flow.md`

Reply:
Done!